python/triton/ops/matmul.py

```diff
@@ -155,33 +156,41 @@ def _call(a, b, dot_out_dtype, allow_tf32, fp8_fast_accum):
         assert a.shape[1] == b.shape[0], "incompatible dimensions"
         M, K = a.shape
         _, N = b.shape
+
+        # common type between a and b
+        ab_dtype = get_higher_dtype(a.dtype if a.dtype not in [tl.float8e4nv, tl.float8e4b15, tl.float8e5] else torch.float16,
```

Comment:
nit: does something like `"float8" in str(a.dtype)` work instead of checking for this list?

Reply:
Also, would it make sense to make these `if`s a part of the logic within `get_higher_dtype`?